tests/test_facebook_invalid_attribution_window.py

```diff
@@ -38,3 +42,18 @@ def test_run(self):
         discovery_error_message = exit_status.get('discovery_error_message')
         # validate the error message
         self.assertEquals(discovery_error_message, "The attribution window must be 1, 7 or 28.")
+        self.assertIsNone(exit_status.get('target_exit_status'))
+        self.assertIsNone(exit_status.get('tap_exit_status'))
+
+
+class FacebookInvalidAttributionWindowStr(FacebookInvalidAttributionWindowInt):
+
+    @staticmethod
+    def name():
+        return "tt_facebook_invalid_window_str"
+
+    def test_run(self):
```

Comment:
unitest has a skip decorator that could be used here to avoid failures until the bug is addressed.

Reply:
done